projects/core/tsconfig.lib.json

```diff
@@ -12,7 +12,7 @@
     ]
   },
   "angularCompilerOptions": {
-    "compilationMode": "partial",
+    "compilationMode": "full",
```

Comment:
My understanding is that we do not want full compilationMode for either module in either build type, as I believe it will create problems for people who aren't using exactly the same version of Angular we released it under.

Reply:
You're right. Reverted.